Java/corenative/src/main/java/com/breadwallet/corenative/crypto/BRCryptoTransferState.java

```diff
@@ -30,21 +30,25 @@
             public long transactionIndex;
             public long timestamp;
             public BRCryptoFeeBasis feeBasis;
+            public boolean success;
+            public String error;
```

Comment:
This should be public byte[] error = new byte[SIZE_GOES_HERE];

Then what I've done in the past (and could be done for `success` too once changed to an int), is have an accessor (`public boolean success()` and `public Optional<String> error()`) that takes the raw field and converts it to the more natural Java type.

Reply:
Perfect.  Thanks.